docs/r3/example_design_for_atomic_swap.md

```diff
@@ -0,0 +1,212 @@
+
+It is customary at this point to introduce names for these participants. However, our experience is that it proves almost impossible for the human brain to keep track of which role each of 'Alice' and 'Bob' are performing once one reaches the level of detail required in a design overview.  And so, in what follows, we try to be more explicit.
+
+We will mostly refer to the 'asset seller' and the 'asset purchaser'. And, where it aids clarity, we will also refer to the participants in terms of which network holds their starting balance. 
+
+In other words: 
+
+* the 'asset seller' is the party which begins by holding value, in the form of a digital asset, on the Corda network, and which will end by holding digital cash on the EVM network.
+* the 'asset purchaser' is the party which begins by holding value, in the form of digital cash, on the EVM network, and which will end by holding a digital asset on the Corda network.
+
+## System Overview
+
+The system consists of two networks, an EVM-based network and a Corda network. Each participant of the swap has a presence on both networks. 
+
+This means that they each have access to a Corda node that they control and they have the ability to sign and send transactions to the EVM network. Importantly, we assume they also each have access to a Web3 RPC endpoint that is operated by a party they trust (which may in fact be themselves)
+
+Finally, we assume that a party's Corda node has the ability (including the necessary network connectivity to connect to their trusted Web3 endpoint. In a production context, it is likely that such a connection would be proxied or subject to complex routing of some sort, but it doesn't change the analysis that follows.
+
+### Capabilities of Corda Nodes
+The Corda nodes can interact with the EVM network through RPC endpoints and Web3 APIs, enabling them to:
+
+1.	Subscribe to EVM events.
+2.	Trigger Corda flows in response to EVM events using filters and collecting event parameters.
+3.	Create EVM transactions, execute EVM contract calls, execute Web3 API calls on the EVM Network RPC endpoint.
+4.	Generate and validate Merkle proofs attesting a certain EVM event belongs to a certain block
+5.  Query for evidence that a particular block has been finalised 
+6. (Optional, to support non-PoA scenarios) Attest a block is final.
+7. (Optional, to support non-PoA scenarios) Query other Corda nodes asking them to attest, by signature over the block header, that (in their view) a given block exists and is final. In this capacity, these queried nodes are known as Oracles.
```

Comment:
dislike the oracle phrasing

Reply:
added a comment saying we may revisit this